src/screen.c

```diff
@@ -21,28 +21,23 @@ void screen_init()
 void screen_clear()
 {
   for (u16 x = 0; x < VGA_WIDTH; x++)
-  {
     for (u16 y = 0; y < VGA_HEIGHT; y++)
-    {
-      display_buffer[screen_position_to_index(x, y)] = '\0' | (7 << 8);
-    }
-  }
+      display_buffer[screen_position_to_index(x,y)]  = '\0' | (0 << 8);
 }
 
 void screen_set_cursor_position(i8 x, i8 y)
 {
-  u16 index = screen_position_to_index(x, y + 1);
+  u16 index = screen_position_to_index(x,y);
```

Comment:
Why is this change necessary exactly? And please stick to the formatting rules of the project, _once_ again.